packages/m2m-gateway/src/services/purposeService.ts

```diff
@@ -98,5 +100,27 @@ export function purposeServiceBuilder(clients: PagoPAInteropBeClients) {
         logger,
       });
     },
+    getPurposeVersion: async (
+      purposeId: PurposeId,
+      versionId: PurposeVersionId,
+      { logger, headers }: WithLogger<M2MGatewayAppContext>
+    ): Promise<m2mGatewayApi.PurposeVersion> => {
```

Comment:
Same from previous PRs
```suggestion
    async getPurposeVersion(
      purposeId: PurposeId,
      versionId: PurposeVersionId,
      { logger, headers }: WithLogger<M2MGatewayAppContext>
    ): Promise<m2mGatewayApi.PurposeVersion> {
```